languageserver/integration/resolving.go

```diff
@@ -78,3 +78,22 @@ func (i *FlowIntegration) getAccount(address common.Address) (*flow.Account, err
 
 	return account, nil
 }
+
+const scriptGetAddress = `
+import AccountManager from 0xSERVICE_ACCOUNT_ADDRESS
+
+pub fun main(name: String): Address? {
+    return AccountManager.getAddress(name)
+}
+`
```

Comment:
Might be a good idea to move this to `templates.go` so it's defined alongside the `AccountManager` contract

Reply:
Yap, forgot about this one. Thank you for discovering it!